spec/Section 3 -- Type System.md

```diff
@@ -2058,8 +2058,8 @@ directive @example on
 
 Directives can also be used to annotate the type system definition language as
 well, which can be a useful tool for supplying additional metadata in order to
-generate GraphQL execution services, produce client generated runtime code, or
-many other useful extensions of the GraphQL semantics.
+generate GraphQL services, produce client generated runtime code, or many other
```

Comment:
(I added this change as I grepped through for execution references, everywhere else we just refer to a "GraphQL service" not a "GraphQL execution service")